src/main/java/org/example/springproject/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package org.example.springproject.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "books")
+@NoArgsConstructor
```

Comment:
```suggestion
```
Remove this annotation, if you don't have any constructor with parameters, default constructor will be generated by default